spec/forms/provider_interface/offer_wizard_spec.rb

```diff
@@ -57,6 +70,59 @@
     end
   end
 
+  describe '.build_from_application_choice' do
+    let(:application_choice) do
+      create(
+        :application_choice,
+        course_option: create(:course_option),
+        offer: offer,
+      )
+    end
+    let(:offer) { { 'conditions' => ['Fitness to train to teach check', 'Be cool'] } }
+
+    it 'correctly populates the wizard with offer conditions' do
+      wizard = described_class.build_from_application_choice(
+        store,
+        application_choice,
+      )
```

Comment:
Consider using a context around all examples and moving the wizard setup in a let block, to follow the pattern of the other examples in this spec.